pkg/config/config_test.go

```diff
@@ -933,18 +933,21 @@ func TestSetConfiguration(t *testing.T) {
 	tests := []struct {
 		name           string
 		parameter      string
+		value          interface{}
 		existingConfig Config
-		want           bool
+		want           interface{}
```

Comment:
Why is there no wantErr in this tests? The function seems to throw error in case of case error and others

Reply:
now add -ve cases as well as wantErr

Reply:
Maybe a -ve case also for extra surity?